### DISCUSSION TITLE
Permission settings for 'Interfaces' under Virtualization menue

### DISCUSSION THREAD

**Discussion by shamim00shahriar** [netbox-community/netbox#5705] [General]
Hello everyone.

I have created a group called localAdmins, and have given permission to each and everything related to interfaces; however, they are still unable to make use of the 'Interfaces' submenu under 'Virtualization'
![image](https://user-images.githubusercontent.com/78094014/106012225-1c96f600-60b3-11eb-9a18-55eb07cd2531.png)

If they click on it, they get Access Denied
![image](https://user-images.githubusercontent.com/78094014/106012356-3cc6b500-60b3-11eb-8974-2bfd524fdc1b.png)

If i make the user a 'SuperUser', then it works. But it makes the system susceptible to someone messing about with admin user and so on (which I definitely wish to avoid). 

Could anyone please point me to the correct setting that I can use to make sure that local admins can use the 'Interfaces' under 'Virtualization'?

The system in question is NetBox v2.9.11 running in a FreeBSD 12x server.

Best regards
Shamim

**Comment by DanSheps** [comment]
What permissions do you have defined?

**Reply by shamim00shahriar** [reply]
Hi DanSheps

anything I could see with 'interface' on it
![image](https://user-images.githubusercontent.com/78094014/106014450-6a146280-60b5-11eb-9ed9-c8fc7a19abf6.png)

As far as I can tell, this should have done the trick, but it does not!

**Comment by candlerb** [comment]
Note that as of Netbox [2.9.0](https://github.com/netbox-community/netbox/releases/tag/v2.9.0), "interfaces" and "vminterfaces" are two different object types (whereas previously they were the same object).

![image](https://user-images.githubusercontent.com/44789/106014632-9203c600-60b5-11eb-9f87-ceb74e3db93c.png)

**Reply by DanSheps** [reply]
Pretty sure this is it here, and is kind of what I was getting at...  ;)

**Comment by shamim00shahriar** [comment]
Thank you both -- issue resolved! I did not realize between the updates the objects changed. I have added the virtualization > interface to the list, and it resolved the issue.

Not sure how to mark it as "resolved" or if there even is a need to do so, but it is resolved for me.

Very much appreciated.
Best regards
